kafka-ballerina/src/kafka/tests/test_utils/kafka_server.bal

```diff
@@ -0,0 +1,33 @@
+import ballerina/system;
```

Comment:
Can we add the license header?

Reply:
Fixed with a2a091b71c90d31b58299a6ee6e51d4dec82081e